packages/web-components/src/components/va-modal/va-modal.tsx

```diff
@@ -349,56 +354,58 @@ export class VaModal {
       : 'Close modal';
 
     return (
-      <Host
-        aria-label={ariaLabel}
-        aria-modal="true"
-        class="va-modal"
-        role={ariaRole(status)}
-      >
-        <div class={wrapperClass} tabIndex={-1}>
-          <button
-            aria-label={btnAriaLabel}
-            class="va-modal-close"
-            onClick={e => this.handleClose(e)}
-            ref={el => (this.closeButton = el as HTMLButtonElement)}
-            type="button"
-          >
-            <i aria-hidden="true" />
-          </button>
-          <div class={bodyClass}>
-            <div role="document">
-              {modalTitle && (
-                <h1 class={titleClass} tabIndex={-1}>
-                  {modalTitle}
-                </h1>
-              )}
-              <slot></slot>
-            </div>
-            {((primaryButtonClick && primaryButtonText) ||
-              (secondaryButtonClick && secondaryButtonText)) && (
-              <div
-                class="alert-actions"
-                ref={el => (this.alertActions = el as HTMLDivElement)}
-              >
-                {primaryButtonClick && primaryButtonText && (
-                  <button
-                    onClick={e => this.handlePrimaryButtonClick(e)}
-                    type="button"
-                  >
-                    {primaryButtonText}
-                  </button>
-                )}
-                {secondaryButtonClick && secondaryButtonText && (
-                  <button
-                    class="button-secondary"
-                    onClick={e => this.handleSecondaryButtonClick(e)}
-                    type="button"
-                  >
-                    {secondaryButtonText}
-                  </button>
+      <Host>
```

Comment:
What are these changes for?

Reply:
The change here is adding a wrapper div element and moving the aria, class and role attributes to that new wrapper. This was needed because the styles from the class `va-modal` were not being applied.